rspace/src/main/scala/coop/rchain/rspace/PointerBlock.scala

```diff
@@ -0,0 +1,31 @@
+package coop.rchain.rspace
+
+import scodec.Codec
+import scodec.Codec._
+import scodec.codecs._
+
+case class PointerBlock(toVector: Vector[Option[Blake2b256Hash]]) {
```

Comment:
In the code?

Reply:
Yes, I have no idea what `None`s represent in this data structure at the moment. Why not for example just use `Vector[(Int, Blake2b256Hash)]` ? Although this would mean that vector has to be reallocated when it resizes.